cedar-policy-generators/src/err.rs

```diff
@@ -46,6 +46,8 @@ pub enum Error {
         /// short string describing what we were doing when this happened
         doing_what: &'static str,
     },
+    /// An error that occurs attempting to convert a hierarchy into an entities structure
+    EntitiesError(String),
```

Comment:
where do we actually construct this variant?

Reply:
Converting a hierarchy into an `Entities` struct in `eval-type-directed`